Mention bind-mounting plugins dir in Docker docs (#82622)
elastic/elasticsearch: docs/reference/setup/install/docker.asciidoc

chgrp 0 esdatadir
--------------------------------------------

You can also run an {es} container using both a custom UID and GID. You

> must ensure that file permissions will not prevent {es} from executing. You
> have two options.

Reviewer: I'd add a semicolon to the end and emphasize you only need to do one of these.
```suggestion
must ensure that file permissions will not prevent {es} from executing. You
can use one of two options:
```

> * Pass the `--group-add 0` command line options to `docker run`. This

Reviewer: I think this is a singular option.
```suggestion
* Pass the `--group-add 0` command line option to `docker run`. This
```

  ensures that the user under which {es} is running is also a member of the
  `root` (GID 0) group inside the container.

> * Alternatively, you must bind-mound the `config`, `data` and `logs`

Reviewer: We can just use the imperative here.
```suggestion
* Bind-mound the `config`, `data` and `logs`
```

>   directories. You must also bind-mount the `plugins` directory if you
>   intend to install plugins and prefer not to <<_c_customized_image, create a
>   custom Docker image>>.

Reviewer: I'd flip the order here so the user's intent comes first.
```suggestion
  directories. If you intend to install plugins and prefer not to
  <<_c_customized_image, create a custom Docker image>>, you must also
  bind-mount the `plugins` directory.
```

>   custom Docker image>>.

Reviewer: Should we move the bind-mounting bullet first? I imagine most admins are using Docker compose in production so I'm not sure if we want to highlight the `docker run` option first.

Author: I'm honestly not sure which option an admin would prefer, but I do suspect that people following the guide to get something up-and-running would be better off using the bind-mounts, so I've swapped the options.

Reviewer 2: >  I imagine most admins are using Docker compose in production so I'm not sure if we want to highlight the docker run option first.

FWIW, I don't think folks using our Docker images in production are using either of these options. They are likely using Kubernetes or some other orchestrator or config as code mechanism to deploy. That's just my intuition though.
